.github/workflows/publish_arm.yml

```diff
@@ -0,0 +1,67 @@
+name: Publish ARM
+
+on:
+  workflow_run:
+    workflows: ["Full Clients Publish"]
+    types:
+      - completed
+
+jobs:
```

Comment:
This needs to run after the build from [here](https://github.com/threefoldtech/tfgrid-sdk-ts/blob/12a427059196245379a567536f3edaa238daadad/.github/workflows/publish.yml#L12). I'm not sure it is the case. It's important because the docker image installs the specified GRID_VERSION (env argument) which might not yet be published when this job starts.

Reply:
These lines trigger the arm build workflow after each completion of the main publish workflow, not before. Below in the 'Get tag' step, we extract the tag that triggered the main publish workflow so we can use it as needed.

Reply:
for the current flow, it will run after the `publish` workflow is completed regardless of its success or not. 
so can we please check if the publish workflow is completed successfully like this [example](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#running-a-workflow-based-on-the-conclusion-of-another-workflow)?

Reply:
I have added the condition of success for the main build workflow.